src/elements/datepicker/datepicker/datepicker.ts

```diff
@@ -33,6 +33,7 @@ export interface SbbInputUpdateEvent {
  * it returns the related SbbDatepickerElement reference, if exists.
  * @param element The element potentially connected to the SbbDatepickerElement.
  * @param trigger The id or the reference of the SbbDatePicker.
+ * @deprecated No longer in use.
```

Comment:
Currently it's still used in tests. maybe possible to remove them right away?

Reply:
I think I would only remove the tests once we remove the function.

Reply:
Absolutely fine :-)